style.css

```diff
@@ -0,0 +1,273 @@
+body{
+    margin: 0;
+    font-family: Roboto, sans-serif;
+}
+
+.main-container{
+    height:100vh;
+    width: 100vw;
+    display: grid;
+    grid-template-columns: 1fr 4fr;
+    grid-template-rows: 1fr 4fr;
+}
+
+.sidebar{
+    padding: 20px 20px 0px 30px;
+    background-color: #0284c7;
+    grid-row: 1/3;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+.sidebar > *{
+    color: white;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.dashboard-content{
+    align-items: center;
+    font-size: 24px;
+    font-weight: 800;
+    display: flex;
+    gap: 10px;
+}
+
+.sidebar img{
+    filter: invert(1);
+    height: 25px;
+}
+
+.dashboard-content img{
+    height:50px
+}
+
+.user-menus,.security{
+    margin-top:20px;
+    font-size: 18px;
+    font-weight: 600;
+}
+
+.dashboard-container ~ div .child{
+    display: flex;
+    gap: 30px;
+    align-items: center;
+}
+
+.header{
+    padding: 5px 40px 5px 40px;
+    box-shadow: 0px 1px 2px 2px #04040487;
+    display: flex;
+    flex-direction: column;
+    gap:10px;
+}
+
+.header > *{
+    flex: 1;
+}
+
+.search-area{
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    align-items: center;
+}
+
+.search{
+    display: flex;
+    align-items: center;
+    gap: 20px;
+}
+
+.search-area .icon{
+    height: 25px;
+}
+
+input{
+    width:50vw;
+    border-radius: 20px;
+    height: 30px;
+    background: #cbd5e1b2;
+    border: 0;
+}
+
+.user-info{
+    display: flex;
+    justify-content: flex-end;
+    gap: 40px;
+    align-items: center;
+}
+
+.search-area img{
+    height: 40px;
+}
+
+.search-area .user-name{
+    font-size: 18px;
+    font-weight: 800;
+}
+
+.header-info{
+    align-items: center;
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    gap: 20px;
+    justify-items: last baseline;
+}
+
+.header-info img{
+    height: 65px;
+}
+
+.user-greeting{
+    gap: 20px;
+    display: flex;
+    align-items: center;
+    justify-self: flex-start;
+}
+
+.btns{
+    display: flex;
+    gap: 20px;
+}
+
+button{
```

Comment:
you can use `shadow` property on button to make it little more clear

Reply:
done